models/fates/DESCRIPTION

```diff
@@ -1,4 +1,4 @@
-Package: PEcAn.FATES
+Package: PEcAn.CTSM-FATES
```

Comment:
@mdietze or @infotroph I believe we don't use hypens or underscore in package names, right? What would you suggest?

Reply:
According to R package naming rules, package names may only contain numbers, letters, and periods, must start with a letter, and can't end in a period. This means package names cannot have hyphens or underscores.   

On our side, there's a small chance that including a period in a model name might break downstream code that builds and parses model-specific function names from the model name in the database / pecan.xml.

Also, much more important, if you're going to change the model name you need to change it EVERYWHERE. This means changing the namespace, changing the individual function names, changing the documentation, changing the records in BETY, etc.

Reply:
Thanks, yes those (changing the namespace/function names/documentation) were going to be my next suggestions. 

But to confirm, do you prefer to keep the new package (this PR) simply as `FATES` or any structurally correct formulation (e.g. `PEcAn.CTSMFATES`) would be OK? I believe @huitang-earth had legitimate concerns to distinguish it with ctsm. In that case, would you prefer that `PEcAn.FATES` package is kept untouched but a new `PEcAn.CTSMFATES` package is added?

Reply:
So if someone wanted to do a coupling to just CTSM without FATES I'd recommend they either start with a new package or do a complete overhaul of the CLM45 model coupler. I don't think a need to distinguish CTSM and CTSM-FATES is a legitimate reason to change the name from FATES to CTSMFATES

In terms of historical legacy, the PEcAn.FATES packages has only ever been coupled to CTSM. We have never tried running it coupled to E3SM. I'd need to see an example of how those runs are set up to determine whether they would use mostly the same functions or would require a complete rewrite. In the latter case, which is what I suspect would happen, we'd probably create a E3SMFATES package.  When we launched the package we had hopes that former case would work (one package for both models), but that planning is 7 years out-of-date at this point.

Overall, I think some changes to the documentation (at the package level and the overall PEcAn documentation) would be a MUCH easier way to make clear that PEcAn.FATES is coupled to CTSM.

Reply:
OK thanks Mike this clarifies, @Hhh-hyc  Yucong will continue this work under `PEcAn.FATES`